lib/psm.c

```diff
@@ -511,13 +511,14 @@ void rpmpsmNotify(rpmpsm psm, int what, rpm_loff_t amount)
  */
 static void markReplacedInstance(rpmts ts, rpmte te)
 {
+    /* this must match rpmNameVersionCompare in depends.c */
```

Comment:
From reading https://bugzilla.redhat.com/show_bug.cgi?id=104066 the nameversioncomprare.diff patch shouldn't be needed anymore, our packages have a SHA1 signature, does that mean we also wouldn't need this patch?